Tests/scripts/slack_notifier.py

```diff
@@ -21,7 +21,7 @@
 SDK_BUILD_TITLE = 'SDK Nightly Build'
 SDK_XSOAR_BUILD_TITLE = 'Demisto SDK Nightly - Run Against Cortex XSOAR'
 BUCKET_UPLOAD_BUILD_TITLE = 'Upload Packs To Marketplace Storage'
-PACKS_RESULTS_FILE = "failed_packs_prepare_content.json"
+PACKS_RESULTS_FILE = "packs_results.json"
```

Comment:
Can you explain this?

Reply:
The shared file that indicates on the successful/failed packs from Prepare Content is `$CIRCLE_ARTIFACTS/packs_results.json`, so there was a mistake here. See [this](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5755c24dc9e77c0001e4cb97/5fb2541c8fc60562af71589c-0-build/artifacts/artifacts/packs_results.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201116T104238Z&X-Amz-SignedHeaders=host&X-Amz-Expires=59&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20201116%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=a5f8fa39a968c06203d0cc4ba243edc15dabd3571a54a72686fc9b2b486faab1) as an example.

Reply:
So it's not that you added a feature to print successful packs, it had a bug in the filename and you print it, right?

Reply:
yes, the feature was merged yesterday